openjdk.test.jck/config/jck8b/compiler.jti

```diff
@@ -0,0 +1,193 @@
+#JavaTest Harness Configuration Interview
```

Comment:
Ad space after # for clarity

Reply:
It's the way the JCK configuration interview tool writes it. It might be more 'clear' if we left it that way.

Reply:
I guess so if that's the reason for it :-)